thinkhazard/static/less/common.less

```diff
@@ -67,6 +68,8 @@ ul.horizontal li {
 
 .page-header {
   position: relative;
+  color: #333;
+  font-weight: bold;
```

Comment:
Instead of setting the font-weight here, I'd suggest we use `variablebold` font for headings.
Currently we use `variableregular` for h1 and h2 which is wrong in my opinion.